clarity/contracts/arkadiko-auction-engine-v1-1.clar

```diff
@@ -671,7 +671,6 @@
     (map-set auctions
       { id: auction-id }
       (merge auction {
-        total-debt-burned: (get total-debt-raised auction),
```

Comment:
Q: can you elaborate why we can remove this?

Reply:
The `total-debt-burned` is updated in the `close-auction` function, which is the only place from which this `extend-auction` function is called.

I have added this line again so we can close this PR. Will create a new PR with additional auction tests.